src/fsharp/vs/service.fsi

```diff
@@ -212,7 +212,7 @@ type internal FSharpCheckFileResults =
     ///    and assume that we're going to repeat the operation later on.
     /// </param>
 
-    member GetDeclarationListInfo : ParsedFileResultsOpt:FSharpParseFileResults option * line: int * colAtEndOfPartialName: int * lineText:string * qualifyingNames: string list * partialName: string * ?hasTextChangedSinceLastTypecheck: (obj * range -> bool) -> Async<FSharpDeclarationListInfo>
+    member GetDeclarationListInfo : ParsedFileResultsOpt:FSharpParseFileResults option * line: int * colAtEndOfPartialName: int * lineText:string * qualifyingNames: string list * partialName: string * allEntities: RawEntity list * ?hasTextChangedSinceLastTypecheck: (obj * range -> bool) -> Async<FSharpDeclarationListInfo>
```

Comment:
The FCS (service.fsi) API should not reveal the type "Item" - it should only reveal "FSharpSymbol".  

At the moment this is relying on the FSharp.Editor InternalsVisibleTo the guts of the compiler. It would be great if we could stop relying on this, or at least avoid increasing the extent of reliance on internals.